src/shared/ui/Stepper/index.tsx

```diff
@@ -15,21 +15,24 @@ export * from './contexts';
 
 export { default as useOnUndo } from './useOnUndo';
 
+export type TPayload = {
```

Comment:
I thought you would use it as a generic. Sorry.
Okay but in this case please rename the type to `StepperPayload`